frontend/src/App.css

```diff
@@ -1,38 +1,3 @@
-.App {
+.homeContent {
```

Comment:
nit: can we make css/html class naming with lower case dash format (ex. `home-content`) instead of the camel format? or it's okay if you want to name all the css classes with camel format! 
dashes seem to more common in code in: https://google.github.io/styleguide/htmlcssguide.html
just for consistency lol